binscripts/rvm-installer.asc

```diff
@@ -1,16 +1,16 @@
 -----BEGIN PGP SIGNATURE-----
 
-iQIzBAABCAAdFiEEYsnl9NowDZSsNhZr4gbCn78E/xcFAliq97YACgkQ4gbCn78E
-/xdnNw//Rh7RX5gGqylBELsn97LjLjfIsNFoTymGeRdvv1ojP4EXwAcBYXRu32+t
-F/v9HjgKqNf4fs27dYFGeXprNkTT1GSZYG47rttkjThxvSdPtjy1SMC/+wiTQ0od
-jNAtjemc3V7tvwk+8sPjyvkIegyhDQNoqKT/2xa4luso9k5Y9n2yf6/2ezlZFHiw
-DyurBeNgik9ROiB69EhD4Vvwnty+wNIA34jKua24nOJSzE8xArxvaE5sY0zUlmeE
-KCAAMKkBM7iwvsMcTrcl/N9ujM4cZNrGjiCeeMEY8BVnPgYk99m5ufFIFQuh12Tn
-g+QYFEf8IgzMLrn1vXgDeiZxq1cKRd88mAG78ZxhxoETSF/6aQAAsqei3f4+oUN1
-BMn1V1Ty3PYz+J7eJJUDdS/+z22ie7LePkkqmnj1AAUyAudUnVe+indOqVkXqwh3
-5WDtaK6hPa7GlWCfN/fsbu/5/qtBQ5Wr4aT3JJxjjYv3OpR5pFP8tG1Wd2CnzK8y
-SwYk5vPUvIwgcio7afZDAgDkQsjxE7aEJwzYUVNBhZo5mAcja9DauU1RAlirTair
-hA35NXgdixjkB2tDdnR5lsm7ggzd0uIjE+RmcIQLNL+uF0iBCM0CS/XcguBlFbCS
-BYO6MW6OlSa5wRM/YfadW6kuPNMYd5Ax+6tpIoB8PheZXdjaP6g=
-=5YX5
+iQIzBAABCAAdFiEEfSuvHPN7E+IGnWlWEFvQ5zlJm9sFAlkjMJsACgkQEFvQ5zlJ
+m9sanQ/8DsI0O9bdK553gef1O/jqqM9UKA4EPKyi5L3NrafhY2gnOofEEQLPNua6
+iNaFJeU/YIerxVLiXmQqF/kVvTifEcOZSCFpQ/uB8GBFfFNmpu0a9YEmCzDzf8Li
+obQrIs5Hyej5XkD5mtOb9qMt9iaW71PgjPEfUkwO2toMrLn1bKyks0tM4b5K8pPO
+DB1NwnKT+YBOgF5FxpoaGA06NlSebMYJhPPQvgXZN4Vd1CNzvWaQuSjideHbSKQI
+YBJkO1LmjHDX/D+X9nL5axH+yvFA5gmvGhtY9izuRn59urpsZqjO/8APj+Dz2JGZ
+TlmiikBE0LjRdcn2Wnu31n4bZMA6SxG4/+5NyaPLGQnkibMDQWC0BUKSk2CoIlF6
+s1kI9dTsehr1V3MruISRvXzRJ0jxY1xLrMFhncHkfJU0kzbY3DsepRTw5CfHcegP
+KT/Q2FjYJ/bLde+Z4US7yoNjqzjzVeFF4oWtGgUD0EFAlUsf9sGs5RcaLGUdcaj8
+o+TL4/oHEzvc9rDMjIsCzvUpuaBLwXg1O6IYbr1BLCGabvjlzGX6khCgZFhXF0q9
+3IQWooLgUvjAFd+lZ+RU6gzt0StBA8bLp0H5cZc7GeBpHwug8VsdEKT6qn0Nojp0
+qln/Ip6s0mDKGUF2xt8vudPPqNI8UclJGavQ2NSnmsPh30KB7Dw=
+=5PU3
```

Comment:
it requires my signature too for quite some time, some people run without updates till they need to change ruby... or not even then, this can happen to be long time